tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -172,7 +172,7 @@
 # Email
 EMAIL_USE_SSL = {{ SMTP_USE_SSL }}
 # Forward all emails from edX's Automated Communication Engine (ACE) to django.
-ACE_ENABLED_CHANNELS = ["django_email"]
+ACE_ENABLED_CHANNELS = ENV_TOKENS.get("ACE_ENABLED_CHANNELS", ["django_email"])
 ACE_CHANNEL_DEFAULT_EMAIL = "django_email"
 ACE_CHANNEL_TRANSACTIONAL_EMAIL = "django_email"
```

Comment:
How come the `ACE_ENABLED_CHANNELS` setting causes an issue in your environment, but not `ACE_CHANNEL_DEFAULT_EMAIL` or `ACE_CHANNEL_TRANSACTIONAL_EMAIL`?

Reply:
@regisb We are working on adding a new ACE channel for push notifications (see https://github.com/openedx/edx-ace/pull/277). We need to extend the list of `ACE_ENABLED_CHANNELS` to include this new channel. However, all email settings should remain unchanged as we are expanding the ace_common app with [new settings](https://github.com/openedx/edx-platform/pull/34971/files#diff-1ed4be7d1a4f0267f287987b3214069590f3a3d5a3cccd5af3e4bee8cf4b3addR30) specifically for configuring mobile push notifications.

Reply:
In that case, why not use the "openedx-common-settings" patch at the bottom of this settings file to override the `ACE_ENABLED_CHANNELS` setting?

Reply:
As this work done as part of extending OeX core, I am expecting that new edx-ace `push notifications` channel will become the default option for many operators, as FCM, the backend that is used for push notification can be used not only for mobile apps but also for other clients including web. And OeX operators would want to enable `push notifications` along with `django_email` channel. Possibility to set those vars from an env will ease configuration, as most operators already patching LMS/CMS env.

Reply:
> as most operators already patching the LMS/CMS env.

This is not what I observe. IMHO the lms/cms.env.yml files are weird relics of a past that is long gone. Settings are typically customised using a Tutor plugin, and it's easier to customise the common Python settings than the env.yml files.

Reply:
Do you suggest to include the clear instructions for the operators in the ace_common app to override `ACE_ENABLED_CHANNELS` with the "openedx-common-settings" patch, so to eliminate any confusion about setting `ACE_ENABLED_CHANNELS` instead of this PR?

Reply:
No, there shouldn't be any tutor-specific instructions in edx-platform apps. Instructions to override the ACE_* settings should suffice. End-users can refer to [this tutorial](ace_common) to learn how to create a tutor plugin.

Soooo I'm suggesting to do nothing at all :)

Reply:
@regisb sorry the link is broken, did you mean https://docs.tutor.edly.io/tutorials/edx-platform-settings.html?

Reply:
Sorry -- yes, that's correct. And thus the plugin creation tutorial: https://docs.tutor.edly.io/tutorials/plugin.html#plugin-development-tutorial